src/StackExchange.Redis/ResultProcessor.cs

```diff
@@ -227,7 +227,14 @@ public virtual bool SetResult(PhysicalConnection connection, Message message, in
             }
             if (result.IsError)
             {
-                if (result.StartsWith(CommonReplies.NOAUTH)) bridge?.Multiplexer?.SetAuthSuspect(new RedisServerException("NOAUTH Returned - connection has not authenticated"));
+                if (result.StartsWith(CommonReplies.NOAUTH))
+                {
+                    bridge?.Multiplexer?.SetAuthSuspect(new RedisServerException("NOAUTH Returned - connection has not authenticated"));
```

Comment:
"has not BEEN authenticated"?

Reply:
Maybe "not yet authenticated"? I worry about confusion with "we can't verify the validity of what you connected to" on phrasing - thoughts?